nodes/srl/srl.go

```diff
@@ -45,6 +45,9 @@ set / system tls server-profile clab-profile trust-anchor "{{ .TLSAnchor }}"
 set / system tls server-profile clab-profile authenticate-client false
 {{- end }}
 set / system gnmi-server admin-state enable network-instance mgmt admin-state enable tls-profile clab-profile
+set / system gnmi-server rate-limit 65000
+set / system gnmi-server commit-save false
```

Comment:
let's remove this, since this is a default

Reply:
done